include/cutlass/gemm/dispatch_policy.hpp

```diff
@@ -293,15 +294,14 @@ struct MainloopSm90ArrayTmaGmmaWarpSpecialized {
 
 
 #if defined(SYCL_INTEL_TARGET)
-struct MainloopIntelPVCBase {
-  constexpr static int Stages = 1;
+template<int Stages_>
```

Comment:
Why do you call this `Stages`? "stage" in cutlass is "a nested level of tiling which corresponds to a layer of concurrency" (https://github.com/NVIDIA/cutlass/blob/main/media/docs/efficient_gemm.md). Prefetch effects a single layer of memory hierarchy and the distance is just the granularity of that level.

I think we should either stick with the previous naming of prefetch distance. Or we could express it as a PrefetchTileShape (the TileShape_ * prefetch-distance in K-dim)

Reply:
You convinced me. We should keep it as-is.